src/entities/observation-entity.ts

```diff
@@ -53,8 +55,57 @@ export enum Verified {
   Rejected = 'rejected',
 }
 
+interface RawObservationBase<TCommon, TRing, TSpecies> {
+  ring: TRing;
+  ringMentioned: string;
+  speciesMentioned: TSpecies;
+  sexMentioned: TCommon;
+  ageMentioned: TCommon;
+  latitude?: number;
+  longitude?: number;
+  photos?: string[];
+  distance?: number;
+  direction?: number;
+  remarks?: string;
+  date?: Date;
+  accuracyOfDate: EntityDto;
+  placeCode: PlaceCodeDto;
```

Comment:
should be `string`

Reply:
Currently its object { id: string, desc: string };
I suggest to update it in separate pr, as we have some another mismatches(as "string | null" that swagger can't parse)